app/src/main/java/com/concordium/wallet/ui/recipient/recipientlist/RecipientListViewModel.kt

```diff
@@ -18,7 +18,7 @@ class RecipientListViewModel(application: Application) : AndroidViewModel(applic
 
     private val allRecipientsLiveData: LiveData<List<Recipient>>
     val recipientListLiveData: LiveData<List<Recipient>>
-        get() = Transformations.switchMap(allRecipientsLiveData) { allRecipients ->
+        get() = allRecipientsLiveData.switchMap { allRecipients ->
```

Comment:
Why was this change necessary?

Reply:
the Transformations.switchMap was deprecated and moved to extension function.